packages/fxa-auth-server/fxa-oauth-server/lib/jwt_access_token.js

```diff
@@ -35,6 +35,10 @@ exports.create = async function generateJWTAccessToken(accessToken, grant) {
     sub: await sub(grant.userId, grant.clientId, grant.ppidSeed),
   };
 
+  if (grant['fxa-subscriptions']) {
+    claims.scope = `${claims.scope} ${grant['fxa-subscriptions']}`;
```

Comment:
Okay, so I tweaked my local configs a bit, subscribed to 123done Pro locally. I can see subscription capabilities appearing in the `scope` of the JWT access token:
```
"scope": "profile 123donePro",
```
Seems kind of weird that these are mingled, IMO: subscription capabilities aren't *exactly* like OAuth scopes. RPs can't request a token with them, we expose them using the intersection of user subscriptions and RP capabilities.

But, I'm also vague on my OAuth-fu - so if this makes pragmatic & standards-wise sense, it's not a thing I'd fuss too much over.

Reply:
I'm going to go back to adding a new claim for subscriptions because [scope checking is very lax](https://github.com/mozilla/fxa/issues/2478). A malicious user who figures out what the value of the subscription is could load up FxA to sign into the proxy, edit the URL to add the subscription name to the scope list, sign in, and the subscription value would end up in the scope list in the JWT sent to the service provider. If we make scope checking more restrictive, including taking into account subscription values, we can re-visit this. I just tried this using local 123done and it works. :(

Thanks for pushing back here @lmorchard.